test/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests/TestProfiledCommand.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Net;
+using StackExchange.Redis;
+using StackExchange.Redis.Profiling;
+
+namespace OpenTelemetry.Instrumentation.StackExchangeRedis.Tests;
+
+internal class TestProfiledCommand(DateTime commandCreated) : IProfiledCommand
+{
+    private readonly DateTime commandCreated = commandCreated;
+    private readonly CommandFlags flags = CommandFlags.None;
+    private readonly EndPoint endPoint = new IPEndPoint(0, 0);
+
+    public TestProfiledCommand(DateTime commandCreated, CommandFlags flags)
+        : this(commandCreated)
+    {
+        this.flags = flags;
+    }
+
+    public TestProfiledCommand(DateTime commandCreated, EndPoint endpoint)
+        : this(commandCreated)
+    {
+        this.endPoint = endpoint;
+    }
+
+    public EndPoint EndPoint => this.endPoint;
+
+    public int Db => 0;
+
+    public string Command => "SET";
+
+    public CommandFlags Flags => this.flags;
+
+    public DateTime CommandCreated => this.commandCreated;
+
+    public TimeSpan CreationToEnqueued => new(0, 0, 1);
```

Comment:
If these TimeSpan properties are not used could you simply initialize them to `default` instead?

```suggestion
    public TimeSpan CreationToEnqueued => default;
```